lib/erl_interface/test/ei_format_SUITE_data/ei_format_test.c

```diff
@@ -190,6 +197,16 @@ TESTCASE(maps)
     send_format("#{ [a] => 1 , 2 => [b,c] , \"3\" => {c,d} }");
     send_format("#{a=>[],[]=>#{2=>d}}");
     send_format("#{ a => [] , [] => #{ 2 => d } }");
+
+    /* Incorrect map syntax */
+    if (send_format("#{a= >1}") == 0) report(0);
```

Comment:
looks like this is very useful addition to syntax tests, and should extend to lists and tuples (in other pr)